src/containers/Tenant/Diagnostics/Consumers/columns/columns.tsx

```diff
@@ -23,7 +29,27 @@ export const columns: Column<IPreparedConsumerData>[] = [
         name: CONSUMERS_COLUMNS_IDS.CONSUMER,
         header: CONSUMERS_COLUMNS_TITILES[CONSUMERS_COLUMNS_IDS.CONSUMER],
         align: DataTable.LEFT,
-        render: ({row}) => row.name || '-',
+        render: ({row}) => {
+            if (!row.name) {
+                return '–';
+            }
+
+            const queryParams = qs.parse(location.search, {
+                ignoreQueryPrefix: true,
+            });
+
+            return (
+                <EntityStatus
```

Comment:
why EntityStatus? Isn't it just a link?

Reply:
This component is used for links almost in every table (Tenants, Nodes, Partitions), so I used it to keep similar styles across the project

Reply:
yea, but in all of those places there is a status icon and a clipboard button
this column only contains link

Reply:
i mean, this is not an entity status, this is a link to a partition, no statuses here
in some of the cases where EntityStatus is being used now, it is also semantically incorrect. The solution is to either re-purpose the component and rename it, or create another one, or extract some functionality from it and create two components that would match all use cases semantically.
The issue I'm pointing at is semantics.

Reply:
but right now there is a component `InternalLink` for example

Reply:
Changed to `Link`